### PR TITLE
259 Run hermes management on already existing kafka brokers

### PR DESCRIPTION
**Pull request by pawelszymczyk**

### CODE REVIEW

**Comment by druminski** [comment]
It would be nice to see integration test for topic creation when it exists in kafka.

**Comment by druminski** [comment]
:+1:

**Comment by pawelszymczyk** [comment]
@druminski I added integration test checking this case, pls check is it ok

**Comment by druminski** [comment]
Looks great, thanks

File: hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/topic/TopicService.java [hunk 1]
```diff
@@ -57,6 +57,15 @@ public void createTopic(Topic topic) {
         topicValidator.ensureCreatedTopicIsValid(topic);
         topicRepository.createTopic(topic);
 
+        if (!multiDCAwareService.topicExists(topic)) {
+            createTopicInBrokers(topic);
+        } else {
+            logger.info(String.format("Skipping creating topic %s on brokers, topic already exists", topic.getQualifiedName()));
```
Comment (adamdubiel) [inline hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/topic/TopicService.java]: `logger.info("Skipping creation of topic {} on brokers, topic already exists", topic.getQualifiedName())`

**Comment by adamdubiel** [comment]
:+1:

File: hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/topic/TopicService.java (hunk 1 above)
Comment (pawelszymczyk) [inline hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/topic/TopicService.java]: done

**Pull request merged**